win32/configure/have_stdckdint.c

```diff
@@ -0,0 +1,42 @@
+/*
+ Check whether stdckdint.h functionality is available.
+ If it's available, then 'have_stdckdint.exe' will be created
+ by config_sh.PL.
+ $Config{i_stdckdint} and the XS symbol I_STDCKDINT will be
+ defined if and only if 'have_stdckdint.exe' prints '0'
+ when executed.
+ Else $Config{i_stdckdint} and I_STDCKDINT will be undef.
+*/
+
+#include <stdio.h>
+#include <stdckdint.h>
+int func_l(long *resultptr, long a, long b) {
+    return (ckd_add(resultptr, a, b) ||
+	    ckd_sub(resultptr, a, b) ||
+	    ckd_mul(resultptr, a, b)) ? 1 : 0;
+}
+
+int func_ll(long long *resultptr, long long a, long long b) {
+    return (ckd_add(resultptr, a, b) ||
+	    ckd_sub(resultptr, a, b) ||
+	    ckd_mul(resultptr, a, b)) ? 1 : 0;
+}
+
+int main(int argc, char **argv) {
```

Comment:
`argc`/`argv` are unused.

```suggestion
int main(void) {
```